src/build.sh

```diff
@@ -94,6 +98,16 @@ function kernel_build()
 
   command="make -j$parallel_cores ARCH=$arch $cross_compile"
 
+  if [ -n "$pkg" ]; then
+    revision=$(cat .version) # Has to be gathered before building
+    pkg_rule="$(pkg_get_rule "$pkg")"
+    if [[ "$?" == 95 ]]; then
+      complain "Package type '$pkg' not supported"
```

Comment:
How about something like this:
"[..] not supported yet. Consider sending a patch for that."